assignments/assignment7/faulty-oops.md

```diff
@@ -0,0 +1,47 @@
+Welcome to Buildroot
```

Comment:
Use markdown formatting for legibility - see how your markdown file is currently formatted at https://github.com/cu-ecen-aeld/assignments-3-and-later-KarthikRevoor/blob/2df8e179db6b953203f6e19b70da30eb84793dcb/assignments/assignment7/faulty-oops.md